lib/spack/spack/test/cmd/compiler.py

```diff
@@ -59,6 +69,81 @@ def mock_compiler_dir(tmpdir, mock_compiler_version):
     return str(tmpdir)
 
 
+@pytest.fixture()
+def mock_two_compiler_dirs(tmpdir, mock_compiler_version):
```

Comment:
Can the name be updated to include more detail about the issue:

`def mock_two_compilers_with_same_spec`